users-auth/auth-router.js

```diff
@@ -7,12 +7,25 @@ router.post('/register', async (req, res, next) => {
 	try {
 		const { username, password, phoneNumber} = req.body
 		const user = await Users.findBy({ username }).first()
+        const phonenumber = await Users.findBy({ phoneNumber }).first()
 
+        if (!username || !password || !phoneNumber) {
+			res.status(401).json({ message: "username, password, and phonenumber are REQUIRED" })
+        }
+
+        if (username.length > 20){
+            res.status(401).json({message: "Username can NOT exceed 20 characters"})
+        }
```

Comment:
Thank you for validating the contestants.
The formatting (indenting) does need to be fixed here as well.